contentcuration/contentcuration/urls.py

```diff
@@ -122,18 +122,9 @@ class ContentKindViewSet(viewsets.ModelViewSet):
 
 
 class ContentNodeViewSet(BulkModelViewSet):
-    queryset = ContentNode.objects.all()
+    queryset = ContentNode.objects.none()
     serializer_class = serializers.ContentNodeCompleteSerializer
 
-    def get_queryset(self):
-        if self.request.user.is_admin:
-            return ContentNode.objects.all()
```

Comment:
Just removing this block would stop the 'list all content nodes' case from working. Then the worst case is 'list all content nodes in channels belonging to user' which is still bad, but far less bad.